spec/tasks/one_off_spec.rb

```diff
@@ -63,4 +63,57 @@
       expect(maven2_version_dependency.platform).to eq("Maven")
     end
   end
+
+  describe "delete_ignored_maven_versions_and_resync_packages" do
```

Comment:
I did not add expectations to ensure the manual_sync is actually run because that behavior is pretty standard, and this spec covers the destructive behavior. If we want to add any checks for that though we can do that.